pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html

```diff
@@ -477,12 +477,12 @@ <h5 i18n="group user search attributes title" aria-level="3">
                             <clr-tooltip>
                                 <clr-icon aria-live="assertive" a11yTooltipTrigger shape="info-circle" size="22"></clr-icon>
                                 <clr-tooltip-content clrPosition="top-left" clrSize="lg" *clrIfOpen>
-                                    <span>The LDAP attribute that Kubernetes will see as part of your groups (e.g., cn).</span>
+                                    <span>The LDAP attribute that Kubernetes will see as part of your groups (e.g., ou). It is searched by OU only for now</span>
```

Comment:
Please change to:

The LDAP attribute that Kubernetes will see as part of your groups (e.g., ou); can only be searched by OU.

Reply:
fixed